README.md

```diff
@@ -8,7 +8,7 @@ __Programming language:__ JavaScript
 __Actions:__ Read, Watch, Buy  
 __Run project:__ https://ruslanpleskun.github.io/cstd-pleskunruslan-2122/  
 __Run tests:__ `npm test`  
-__GIT TAG:__ `Streaming_Service_v2.0_21133`  
+__GIT TAG:__ `StreamingService_v1.1_14_21143`  
```

Comment:
should be: StreamingService_v1p1_WW21483

Reply:
Done